src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller_Commit.cs

```diff
@@ -135,8 +135,8 @@ private void Commit(CompletionItem item, TextChange textChange, Model model, cha
                 }
             }
 
-            // Let the completion rules know that this item was committed.
-            GetCompletionRules().CompletionItemCommitted(item);
+            // Ensure that the item is added to the completion service's most recently used list.
+            GetCompletionService().MostRecentlyUsedList.AddItem(item);
```

Comment:
I'm trying to rationalize the MRU stuff because it's currently baked in a bit wonkily. As an abstraction, it's pretty leaky right now. Still working on it if you have suggestions.